airbyte-integrations/connectors/source-zendesk-talk/unit_tests/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "source-zendesk-talk-tests"
+version = "1.2.0-rc.1"
+description = "Unit tests for source-zendesk-talk"
+authors = ["Airbyte <contact@airbyte.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.10,<3.13"
+airbyte-cdk = "^6.10.0"
```

Comment:
Let's re-lock, this should go to actual 6.31, and this _should_ work. Nothing in the code screams at me as the reason for why requests_mock would not be importable.